docs/source/tools/mythxvsc/index.rst

```diff
@@ -0,0 +1,61 @@
+.. meta::
+   :description: MythXvsc is an extension for running MythX smart contract analysis from your favourite IDE.
+
+.. _tools.mythxvsc:
+
+MythXvsc
+============================
+
+MythXvsc is an extension for running MythX smart contract analysis from your favourite IDE.
```

Comment:
Again, unless I'm misunderstanding, the point here is that this is a VS Code-specific plugin. Let's call it that.